### PR TITLE
Implement `top` option in queryFTS

### PR DESCRIPTION
**Pull request by acquamarin**
This PR adds the`top` optional parameter to `queryFTS` function. The `top` parameter chooses the `top k` result from the queryFTS output.

Performance improvement:
Query:
```
CALL query_fts_index('doc', 'contentIdx', 'dispossessed meaning', top:=topK) RETURN node,score order by score desc;
```

Result:
4 threads:

| topk  | Master | This branch |
|-------|------------------|-----------------|
| 50    | 2.808s           | 1.398s          |
| 500   | 2.977s           | 1.496s          |
| 5000  | 2.906s           | 1.728s          |

### CODE REVIEW

File: extension/fts/test/test_files/ms_passage_small.test [hunk 1]
```diff
@@ -22,6 +22,14 @@
 134|1.923814
 109|1.957072
 390|2.937742
+-LOG QueryKeywordsWithTOP5
+-STATEMENT CALL query_fts_index('doc', 'contentIdx', 'dispossessed meaning', top:=5) RETURN node.id, score;
```
Comment (andyfengHKU) [inline extension/fts/test/test_files/ms_passage_small.test]: Add test for negative, non-integer and exceed-max-num-nodes top parameter.

**Review by andyfengHKU — APPROVED** [review]
